jaeger/charts/linkerd-jaeger/templates/namespace-metadata-rbac.yaml

```
@@ -76,3 +77,4 @@ subjects:
 - kind: ServiceAccount
   name: namespace-metadata
   namespace: {{.Release.Namespace}}
+{{-end}}
```

You require spaces here or else the template breaks, as you can see in the integration tests results.
```suggestion
{{- end }}
```

---

Thanks  @alpeb . Added spaces between those end statements.

---

Thanks, can you address my comment about naming above? After that, this PR looks fine from my perspective.